.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -43,6 +43,12 @@ _Do not alter or remove anything below. The following sections will be managed b
 
 * <!-- One or more bullet points for how to technically resolve the issue. -->
 
+### Test Coverage
+* <!-- One or more bullet points for how to implement automated tests to verify the feature works. -->
+
+### Visual Regression Changes
+* <!-- One or more bullet points describing how the feature will affect visual regression tests, if applicable. -->
```

Comment:
This is a nitpick for sure, but would you please include a blank line between the heading and the bullet for consistency with the other sections? (Applies to all changes here)

Reply:
Blank line added @aaemnnosttv although it was confusing for me at first when using the templates. I didn't know whether I had to add something between the heading and the comment.